mantidproject/mantid
# Change of maximum height of custom dialog for GetNegMuMuonicXRD

The custom dialog for GetNegMuMuonicXRD algorithm should now be big enough to display the whole periodic table when the dialog is first loaded.

**To Test**:
- Load `GetNegMuMuonicXRD` algorithm dialog
- See that the periodic table is visible and you can make the dialog scale bigger.

Fixes #14351
